Updated HDF5 tools page to provide more detailed listing of tools (#85)
NeurodataWithoutBorders/nwb-overview: docs/source/tools/hdftools/hdftools.rst

.. _analysistools-hdftools:

HDF Tools
---------

.. short_description_start

> There are a broad range of useful tools for inspecting and browsing HDF5 files. For example,

Author: ```suggestion
:ref:`analysistools-hdftools`: There are a broad range of useful tools for inspecting and browsing HDF5 files. For example,
```

.. _analysistools-hdftools:

HDF Tools
---------

.. short_description_start

> :ref:`analysistools-hdftools` as part of the HDF5 suite and the broader HDF community
> provide a broad range of useful tools for inspecting and browsing HDF5 files. For example

Reviewer: ```suggestion
There are a broad range of useful tools for inspecting and browsing HDF5 files. For example,
```

Author: Thanks for the suggestion. We need the ``:ref:`analysistools-hdftools`` in the text as this how we navigate from the page with listing of all the tools to the individual pages. I'll see what I can do to smooth out that sentence without loosing the link. I think maybe we can just use your sentence and "HDF5 Tools: " as a title prefix.
